<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookDialog</class>
 <widget class="QDialog" name="AddBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>202</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelBookId">
      <property name="text">
       <string>Book ID:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditBookId"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelBookTitle">
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditBookTitle"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelBookAuthor">
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditBookAuthor"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelBookYear">
      <property name="text">
       <string>Release Year:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditBookYear"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelBookCopies">
      <property name="text">
       <string>Copies Amount:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditBookCopies"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddBookDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddBookDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
